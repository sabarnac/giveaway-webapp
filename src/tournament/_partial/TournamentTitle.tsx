import { inject } from "mobx-react";
import { withTranslation } from "react-i18next";
import TournamentTitle from "./TournamentTitle.react";

/**
 * React component for the tournament title.
 */
export default withTranslation()(inject("config")(TournamentTitle));
