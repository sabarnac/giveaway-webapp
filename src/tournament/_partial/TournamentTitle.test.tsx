import React from "react";
import TournamentTitle from "./TournamentTitle.react";
import { create, ReactTestRenderer } from "react-test-renderer";
import {
  createDummyConfig,
  createDummyTranslationProps,
} from "../../util/test";

describe("TournamentTitle Component.", () => {
  it("Matches snapshot.", () => {
    const component: ReactTestRenderer = create(
      <TournamentTitle
        config={createDummyConfig()}
        {...createDummyTranslationProps()}
      />,
    );

    expect(component.toJSON()).toMatchSnapshot();
  });
});
