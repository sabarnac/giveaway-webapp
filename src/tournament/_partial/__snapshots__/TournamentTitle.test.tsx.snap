// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`TournamentTitle Component. Matches snapshot. 1`] = `
<h1>
  tournamentView.name
   Tournament
</h1>
`;
